contracts/strategies/allocation/quadratic-voting-NFT-tiered/README.md

```diff
@@ -0,0 +1,46 @@
+Spec: NFT-Gated Quadratic Voting (QV) with Tiered Allocation
+---------------------------------
+
+## Overview 
+Quadratic Voting is a popular method for democratic decision-making. This strategy gates voting using NFTs, and allows holders of different NFTs to have different amounts of voice credits. The pool's funds are distributed proportionally to the number of votes each project receives, with caps on the maximum amount of the pool that one recipient can receive. 
+
+## Spec
+### Recipient logic
+In this strategy, prospective recipients need to apply and be approved by a number of admins dictated by the pool. The pool admins can set a time window in which applications must be received.
+- `registerRecipient` - function for recipient to submit application
+    - if a recipient submits an application that meets all requirements, then the recipient status is set to `Pending`
+    - **Customizable Settings**
+        - pool admin can require recipients to have Allo registry identity in order to apply
+            - if pool admin doesn't require Allo identity and the recipient doesn't have one, then `msg.sender` is used as recipient id
+        - pool admin can include optional and required questions in an application, stored in metadata
+            - recipient must answer questions as specified by pool admin
+- `reviewApplication` - function for pool admins to approve or reject a recipient for inclusion in the round.
+    - the pool admins have the ability to set how many admins must confirm a decision in order for an application to be approved or rejected.
+        - For example, if the threshold is set to `2`, then two separate admins must approve or reject an application before its status updates. As soon as the recipient receives either two rejections or two approvals, admins can no longer take any action on that application. 
+
+### Voter eligibility logic
+Pool admins are able to set multiple NFTs as their voting gates. As long as a wallet holds at least one of those NFTs they are marked as `eligible`. 
```

Comment:
We should use `As long as a wallet holds at least one of those NFTs ...` as an example. Some admins may want to use more than one or a combination of them.